frontend/src/components/eam-create-worker/eam-create-worker.tsx

```diff
@@ -0,0 +1,59 @@
+import { InputType, ButtonType } from 'common/enums/enums';
+import { useAppForm } from 'hooks/hooks';
+import { getNameOf } from 'helpers/helpers';
+import { Table, Input, Button } from 'components/common/common';
+import { EAMCreateWorkerRequestDto } from 'common/types/types';
+import { EAMCreateWorker as CreateWorkerValidationSchema } from 'validation-schemas/validation-schemas';
+
+const EamCreateWorker: React.FC = () => {
+  const { control, errors, handleSubmit } =
+    useAppForm<EAMCreateWorkerRequestDto>({
+      defaultValues: {
+        name: '',
+      },
+      validationSchema: CreateWorkerValidationSchema,
+    });
+
+  const onSubmit = (payload: EAMCreateWorkerRequestDto): void => {
```

Comment:
```suggestion
  const hanldeFormSubmit = (payload: EAMCreateWorkerRequestDto): void => {
```
`handle` for locals component's cbs 
`on` for props component's cbs

Reply:
sorry, I pledged github and there were no these messages yesterday and only now I saw